src/libraries/System.Data.DataSetExtensions/Directory.Build.props

```diff
@@ -1,8 +1,6 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <!-- AssemblyVersion is frozen to that which originally shipped in 2.1 -->
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
This library only ships inside the Microsoft.NETCore.App shared framework. I assume we don't need to pin its assembly version anymore?

Reply:
I think it once had a package and this is left over from that time.

Reply:
What is the expected side-effect of removing AssemblyVersion from here?

Reply:
The assembly will version in the same way as every other assembly that ships only in the targeting packs (N.0.0.0 which remains frozen in servicing).